pkg/image/api/types.go

```diff
@@ -157,3 +205,75 @@ type DockerImageReference struct {
 	Tag       string
 	ID        string
 }
+
+// ImageStreamImport allows a caller to request information about a set of images for possible
+// import into an image stream, or actually tag the images into the image stream.
+type ImageStreamImport struct {
+	unversioned.TypeMeta
+	// ObjectMeta must identify the name of the image stream to create or update. If resourceVersion
```

Comment:
Is the `resourceVersion` required to match anything or can I simply set it to "foo"?